main_game.py

```diff
@@ -0,0 +1,174 @@
+import sys
+import pygame
+import random
+import json
+from mechanics.input_processing import generate_response
+from collections import defaultdict
+from settings.game_settings import GameSettings
+from characters.player.player import Player
+from characters.npc.npc import Npc
+from pathlib import Path
+
+image_folder_path = Path.cwd() / "images"
+settings_folder_path = Path.cwd() / "characters" / "npc" / "npc_settings.json"
+
+
+CHARACTER_NAMES = [
+    "npc1",
+    "npc2",
+    "npc3",
+    "npc4",
+    "npc5",
+    "npc6",
+    "npc7",
+    "npc8",
+    "npc9",
+    "npc10",
+    "npc11",
+    "npc12",
+    "npc13",
+    "npc14",
+]
+ATTRIBUTES = [
+    "Intelligence",
+    "Honesty",
+    "Patience",
+    "Humor",
+    "Ambition",
+    "Affection",
+    "Confidence",
+    "Adventurousness",
+    "Empathy",
+]
+COORDINATES = [
+    (0, 0),
+    (0, 256),
+    (0, 512),
+    (256, 0),
+    (256, 256),
+    (256, 512),
+    (512, 0),
+    (512, 256),
+    (768, 0),
+    (768, 256),
+    (768, 512),
+]
+PICKED_COORDINATES = []
+
+
+def generateNPC():
+    global PICKED_COORDINATES
+
+    attribute_dict = defaultdict(int)
+    npc_settings = defaultdict(str)
+
+    npc_name = random.sample(CHARACTER_NAMES, 1)[0]
+    npc_settings["name"] = npc_name
+    npc_settings["image_path"] = str(image_folder_path / "heart.png")
```

Comment:
this image is gone it seems: `FileNotFoundError: No such file or directory: '/Users/pybob/code/pdm/gerardo/project_G/images/heart.png'.`

Reply:
Yeah, when I was testing pyinstaller it required to be in the root of the tree so I copied it, but it didn't work. Forgot to delete again.

Reply:
ah of course, that makes sense, thanks